src/main/java/HW_2/home_task_from_metanit/App.java

```diff
@@ -0,0 +1,75 @@
+package HW_2.home_task_from_metanit;
```

Comment:
Please remove this extra internal folder
![Screenshot_7](https://github.com/AntBelyi/ALevel_automation/assets/66726924/97805da1-607f-4e47-8d12-ec6c3f042941)

Reply:
fixed